app/content/survey-manual.md

```diff
@@ -0,0 +1,261 @@
+newsletter configuration. These ids will then be used in subsequent javascript api calls.
+
+```json
+[
+  {
+    name: 'Basic survey',
+    configurationId: '49c12712-9750-4571-8c67-96722561c13a',
+  },
+  {
+    name: 'Multi-page survey',
+    configurationId: 'e98a2efb-d009-43c9-99ef-dda11c8c5a7f',
+  },
+]
+```
+
+### Invoke Survey Prompts
+
+To configure survey prompts, `swg.js` must first be configured on the page.
+These examples show using the initialization of the library in `manual` mode, but the
+APIs are also available in automatic mode. 
+
+### Get the prompt instance to display
+
+To invoke a survey prompt, a publisher must use the `configurationId` provided by
+Google in response to submitting a prompt configuration. Publishers use the
+`configurationId` to fetch a valid prompt instance using the `subscriptions.getAvailableInterventions()` method from the initialized `swg.js` library.
+
+```javascript
+const publisherConfiguration = {
+  name: 'Basic survey',
+  configurationId: '49c12712-9750-4571-8c67-96722561c13a',
+};
+
+const availableInterventions = await subscriptions.getAvailableInterventions();
+
+const prompt = availableInterventions.find(({configurationId}) => {
+    return configurationId === publisherConfiguration.configurationId;
+});
+```
+
+### Show the prompt
+
+To display a prompt, use the returned value from `subscriptions.getAvailableInterventions()` and use the `show` method:
+
+```javascript
+prompt?.show({
+    isClosable: true,
+    onResult: (result) => {
+        //Store the result, which is the email of the newsletter signup.
+
+        //Return true to let Google know that you have received and processed
+        //the returned email.
+        return true;
+    }
+});
+```
+
+### Handle the response
+
+The `onResult` callback will include information on the configuration used
+to create the prompt, as well as the newsletter subscriber's information. The `configurationId` matches the `configurationId provided to the publisher from Google, in response to the configuration authored by the publisher in the initial step.
+
+!!! note `onResult()` callback format is variable.
+The contents of a prompt determines the shape of the `onResult` callback data.
+For example, a single-page survey response will differ in shape from a multi-page survey.
+For best results, store the entire result.
+!!!
+
+#### Complete Example
+
+This complete example accomplishes the following:
+
+1. Initializes `swg.js` library in manual mode.
+2. When the library is ready, use the `49c12712-9750-4571-8c67-96722561c13a` configurationId to request a prompt to display.  
+3. When the button is clicked, display the prompt.
+4. Store the results of a successful prompt with the sample `PromptPersistence()` library.
+
+!!! note `PromptPersistence()` is an example implementation.
+In a production environment, a publisher would use the prompt response to send data to their own
```

Comment:
"to their own account" -> "to their Google Analytics account"? I don't think newsletters are sent to GA, but they can be downloaded in Publisher Center

Reply:
Added a clarification re: CRM as the destination.